File: algorithm/src/main/java/baekjoon/p02864/Main.java
package baekjoon.p02864;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        for (int solve : solve(sc.nextInt(), sc.nextInt())) {
            System.out.print(solve + " ");
        }
    }

    private static int[] solve(int a, int b) {
        return new int[]{
                convert(a, 6, 5) + convert(b, 6, 5),
                convert(a, 5, 6) + convert(b, 5, 6),
        };
    }

    private static int convert(int n, int from, int to) {
        StringBuilder sb = new StringBuilder();

        for (char ch : String.valueOf(n).toCharArray()) {
            if (ch == ('0' + from)) {
                sb.append(to);
            } else {
                sb.append(ch);
            }
        }

        return Integer.parseInt(sb.toString());
    }

}

File: algorithm/src/main/java/baekjoon/p02864/Main.kt
package baekjoon.p02864

fun main(args: Array<String>) {
    val inputs = readLine()!!.split(" ").map { it.toInt() }
    println(solve(inputs[0], inputs[1]).joinToString(" "))
}

private fun solve(a: Int, b: Int): Array<Int> {
    return arrayOf(
            replace(a, 6, 5) + replace(b, 6, 5),
            replace(a, 5, 6) + replace(b, 5, 6)
    )
}

private fun replace(n: Int, from: Int, to: Int): Int {
    return n.toString().replace('0' + from, '0' + to).toInt()
}